app/Event/ProductionServiceProvider.php

```diff
@@ -30,7 +30,11 @@ function ($app) {
 
         $app[ProductionCommandHandler::class] = $app->share(
             function ($app) {
-                return new ProductionCommandHandler($app[ProductionRepository::class], $app[SimilaritiesClient::class]);
+                return new ProductionCommandHandler(
+                    $app[ProductionRepository::class],
+                    $app[SimilaritiesClient::class],
+                    $app{'event_jsonld_repository'}
```

Comment:
On several other places I see `$app['event_jsonld_repository']`

Reply:
Wow yeah, that's a nice typo there 🙈